TutorLizard.BusinessLogic/Data/ITutorDataAccess.cs

```diff
@@ -1,5 +1,13 @@
-﻿namespace TutorLizard.BusinessLogic.Data;
+﻿using TutorLizard.BusinessLogic.Models;
+
+namespace TutorLizard.BusinessLogic.Data;
 
 public interface ITutorDataAccess
 {
+    Ad CreateAd(int tutorId, string subject, string title, string description);
+    ScheduleItem CreateScheduleItem(int adId, DateTime dateTime);
+    Ad? GetAdById(int adId);
+    int? GetAllAdsId();
+    ScheduleItem? GetScheduleItemById(int? scheduleItemId);
+    int? GetScheduleItemId();
```

Comment:
Like in PR #15, remove unused `GetScheduleItemId` and implement interface in `DataAccess` leaving it with `throw new NotImplementedException();`

Reply:
You added `GetAllAdsId` that you removed in PR #15.